dependencies/pom.xml

```diff
@@ -122,6 +122,9 @@
         <version.lib.mongodb.reactivestreams>1.11.0</version.lib.mongodb.reactivestreams>
         <version.lib.mssql-jdbc>8.4.1.jre8</version.lib.mssql-jdbc>
         <version.lib.neo4j>4.2.4</version.lib.neo4j>
+        <version.lib.hibernate-validator>7.0.2.Final</version.lib.hibernate-validator>
+        <version.lib.validation-api>2.0.1.Final</version.lib.validation-api>
+        <version.lib.el>3.0.0</version.lib.el>
```

Comment:
I think versioning should be handled here. The only jakarta based dependency is updated hibernate-validator now. Others are still javax